docs/fan.md

```diff
@@ -9,14 +9,16 @@ nav_order: 4
 
 ## [](#header-2)Overview
 
-Fans are simple representations of KNX controlled fans. They support setting the speed.
+Fans are simple representations of KNX controlled fans. They support setting the speed and the oscillation.
 
 ## [](#header-2)Interface
 
 - `xknx` XKNX object.
 - `name` name of the device.
-- `group_address` is the KNX group address of the fan device. Used for sending.
-- `group_address_state` is the KNX group address of the fan state. Used for updating and reading state.
+- `group_address` is the KNX group address of the fan speed. Used for sending.
+- `group_address_state` is the KNX group address of the fan speed state. Used for updating and reading state.
+- `group_address_oscillation` is the KNX group address of the oscillation. Used for sending. *DPT 1.001*
+- `group_address_oscillation_state` is the KNX group address of the fan oscillation state. Used for updating and reading state. *DPT 1.001*
 - `device_updated_cb` awaitable callback for each update.
 - `max_step` Maximum step amount for fans which are controlled with steps and not percentage.
```

Comment:
Please add a note like "if `None` percentage is used" and "Default: None" so it is clear when DPT 5.001 or 5.010 is used.